components/org.wso2.carbon.identity.oidc.session/src/test/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServletTest.java

```diff
@@ -0,0 +1,242 @@
+                        "7893-090907ce-eab0-40d2", false},
+                {opbsCookie, true, redirectUrl[2], "failed", " ", null, true,
+                        "7893-090907ce-eab0-40d2", false},
+                {opbsCookie, true, redirectUrl[5], "oauth2_logout_consent.do", " ", null, false,
+                        idTokenHint, false},
+                {opbsCookie, true, redirectUrl[5], "oauth2_logout_consent.do", " ", null, true,
+                        idTokenHint, false},
+        };
+    }
+
+    @Test(dataProvider = "provideDataForTestDoGet")
+    public void testDoGet(Object cookie, boolean sessionExists, String redirectUrl, String expected, String consent,
+                          String sessionDataKey, boolean skipUserConsent, String idTokenHint,
+                          boolean isJWTSignedWithSPKey) throws Exception {
+        TestUtil.startTenantFlow("carbon.super");
+
+        mockStatic(OIDCSessionManagementUtil.class);
+        when(OIDCSessionManagementUtil.getOPBrowserStateCookie(request)).thenReturn((Cookie) cookie);
+        when(OIDCSessionManagementUtil.getErrorPageURL(anyString(), anyString())).thenReturn(redirectUrl);
+
+        mockStatic(OIDCSessionManager.class);
+        when(OIDCSessionManagementUtil.getSessionManager()).thenReturn(oidcSessionManager);
+        when(oidcSessionManager.sessionExists(OPBROWSER_STATE)).thenReturn(sessionExists);
+
+        when(request.getParameter("consent")).thenReturn(consent);
+        when(request.getHeaderNames()).thenReturn(Collections.enumeration(Arrays.asList(new String[]{"cookie" })));
+        when(request.getHeader("COOKIE")).thenReturn("opbs");
+
+        doThrow(new ServletException()).when(commonAuthenticationHandler).doPost(request, response);
+
+        when(request.getSession()).thenReturn(httpSession);
+        when(httpSession.getMaxInactiveInterval()).thenReturn(2);
+
+        mockStatic(IdentityConfigParser.class);
+        when(IdentityConfigParser.getInstance()).thenReturn(identityConfigParser);
+
+        when(request.getParameter("sessionDataKey")).thenReturn(sessionDataKey);
+
+        mockStatic(OAuthServerConfiguration.class);
+        when(OAuthServerConfiguration.getInstance()).thenReturn(oAuthServerConfiguration);
+        when(oAuthServerConfiguration.getOpenIDConnectSkipeUserConsentConfig()).thenReturn(skipUserConsent);
+
+        when(request.getParameter("id_token_hint")).thenReturn(idTokenHint);
+
+        when(OIDCSessionManagementUtil
+                .removeOPBrowserStateCookie(any(HttpServletRequest.class), any(HttpServletResponse.class)))
+                .thenReturn((Cookie) cookie);
+
+        when(OIDCSessionManagementUtil.getOIDCLogoutConsentURL()).thenReturn(redirectUrl);
+
+        when(OIDCSessionManagementUtil.getOIDCLogoutURL()).thenReturn(redirectUrl);
+
+        mockStatic(IdentityTenantUtil.class);
+        when(IdentityTenantUtil.getTenantId(anyString())).thenReturn(-1234);
```

Comment:
it would be better to define a constant for tenant id -1234 & use it

Reply:
or use the relevant constant for -1234 from MultiTenantConstants